e2e/specs/edit-patient.spec.ts

```diff
@@ -25,6 +24,7 @@ const formValues: PatientRegistrationFormValues = {
   stateProvince: 'Pernambuco',
   cityVillage: 'Recife',
   phone: '5555551234',
+  email: 'johnnyronny@example.com',
```

Comment:
There isn't a step in the tests that fill in the email, so adding this is unnecessary.